.github/pull_request_template.md

```diff
@@ -7,17 +7,11 @@ If your pull request closes a GitHub issue, then set its number below.
 
 Fixes # (issue)
 
-## Type of change
-
-Please mark the options that are relevant.
-
-- [ ] Added/Modified tutorials
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
 ## Checklist:
 
 * [ ] I have added tests for my changes
```

Comment:
Q: Could you also remove this.
Since there are changes which might not require this thing :D

Reply:
Sure !